File: 백준/Bronze/10810. 공 넣기/공 넣기.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');
const [N, M] = input[0].split(' ').map(Number);
const steps = input.slice(1);
const baskets = new Array(N).fill(0);

for (let step of steps) {
    const [i, j, k] = step.split(' ').map(Number);
    
    for (let b = i - 1; b < j; b++) {
        baskets[b] = k;
    }
}

const answer = baskets.join(' ');

console.log(answer);

File: 백준/Bronze/10810. 공 넣기/공 넣기.py
import sys

N, M = map(int, input().split())
baskets = [0 for x in range(N)]

for line in sys.stdin:
    i, j, k = map(int, line.rstrip().split())
    
    for idx in range(i-1, j):
        baskets[idx] = k

print(' '.join(map(str, baskets)))